BeeKit/Managers/GoalManager.swift

```diff
@@ -72,19 +72,20 @@ public actor GoalManager {
     public func refreshGoal(_ goalID: NSManagedObjectID) async throws {
         let goal = try modelContext.existingObject(with: goalID) as! Goal
 
-        let responseObject = try await requestManager.get(url: "/api/v1/users/\(currentUserManager.username!)/goals/\(goal.slug)?datapoints_count=5", parameters: nil)
+        let responseObject = try await requestManager.get(url: "/api/v1/users/\(goal.owner.username)/goals/\(goal.slug)?datapoints_count=5", parameters: nil)
```

Comment:
This change seems fine and we can merge with it in, though I'll once again request to minimize unrelated cleanup (there are no feature related changes to this file) in feature PRs.

Reply:
removed from this PR